README.md

```diff
@@ -32,14 +32,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+DOM stands for "Display Object Model". It is an object representation of the HTML of a website. In the DOM, we can change, style and manipulate the page and it's contents.
```

Comment:
To expand on the above a bit you can add that the DOM (Document Object Model) serves as an intermediary between the window object and the javascript one would like to use to manipulate elements on the window object.